dept44-starter/src/main/java/se/sundsvall/dept44/configuration/ObjectMapperConfiguration.java

```diff
@@ -1,10 +1,13 @@
 package se.sundsvall.dept44.configuration;
 
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
 import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
 import com.fasterxml.jackson.module.jaxb.JaxbAnnotationModule;
-import org.springframework.context.annotation.Bean;
 
+@Configuration
```

Comment:
Not sure that this annotation should be here, since `ObjectMapperConfiguration` is included as an auto-configuration import in ´META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports`

Reply:
Yes, I was unsure too. 
Removed it again.